### ISSUE TITLE
Incorrect code created when running a barchart with SASPy

### ISSUE THREAD

**Issue by Cazziii** [sassoftware/saspy#95]
On running the following code:
dataset.bar('gender', title='Title1')

Results are as follows.  The title is removed for the chart due to the additional title ""; added at the end of the code:
proc sgplot data=mydata._df;
	vbar gender ; ;
	title "Title1";
title "";run;

**Comment by tomweber-sas** [comment]
Good catch. Yes, that extra title (to reset it) should have been after the 'run;' I've fixed this and checked the other methods in sasbase.py too. They are all good now, re-setting title after the proc runs. Also, I sae that some had 'title"";' which actually causes a glitch in the output in jupyter; there's a little mark (can't tell what character it's printing), where the title would be instead of nothing there at all. I've changed all of these re-setting title statements to be 'title;' without the quotes (null string) and that has cleaned up that little glitch. So now there should be no title at all unless you specify one.
I've pushed this to master; give it a go and let me know if there's anything else!

Thanks,
Tom
